### PR TITLE
docs: add User Story in capacity-scheduling

### PR DESCRIPTION
**Pull request by yuchen-sun**
I wrote some of the user stories mentioned in #132 
hello, @denkensk  
Can you help me to see if it is right?

### CODE REVIEW

**Comment by denkensk** [comment]
@yuchen-sun  Thanks  
I will review it

File: kep/9-capacity-scheduling/README.md [hunk 1]
```diff
@@ -262,6 +262,37 @@ guaranteed and hence non-preemptable.
...
+are within the reserved "min" quota range.
+
+![5](./5.png)
+
+Next,UserA consumes 4 GPUs reaching the "max" quota of QuotaA.UserB consumes 3 GPUs.UserC still consumes 3 GPUs.
+
+![6](./6.png)
+
+Later,UserB submits a pod that requests 1 GPU, at this time the cluster does not have enough resources and needs 
+to trigger a preemption. When `quota.used + preemptor.request <= min`, victims will be selected from the EQ which used > min
```
Comment (denkensk) [inline kep/9-capacity-scheduling/README.md:281]: `quota.used + preemptor.request <= min` ==> `quotaB.used + preemptor.request <= quotaB.min`

File: kep/9-capacity-scheduling/README.md [hunk 2]
```diff
@@ -262,6 +262,37 @@ guaranteed and hence non-preemptable.
...
+In this case,`QuotaA.used > min`and `QuotaB.used <= min`.So a pod in QuotaA will be the victim.
```
Comment (denkensk) [inline kep/9-capacity-scheduling/README.md:282]: `QuotaB.used <= min` ==> `QuotaC.used <= min`

File: kep/9-capacity-scheduling/README.md [hunk 3]
```diff
@@ -262,6 +262,37 @@ guaranteed and hence non-preemptable.
...
+
+![7](./7.png)
+
+At some point,UserA consumes 2GPUs and UserC consumes 3 GPUs.UserB's two pods consume a total of 5 GPUs.
+
+![8](./8.png)
+
+Next,UserB submits a TaskC that requests 1 GPU, at this time the cluster does not have enough resources and needs
+to trigger a preemption. When `quota.used + preemptor.request > min`, victims will be selected from the same Quota. So TaskA will be the victim.
```
Comment (denkensk) [inline kep/9-capacity-scheduling/README.md:291]: `quota.used + preemptor.request > min` ==> `quotaB.used + preemptor.request > quotaB.min`

File: kep/9-capacity-scheduling/README.md [hunk 4]
```diff
@@ -262,6 +262,37 @@ guaranteed and hence non-preemptable.
 In the end, UserA consumes 4 GPUs and UserB consumes 6 GPUs, satisfying each 
 user's "min" quota.
 
+#### Story 2
+We assume three elastic quotas are defined: QuotaA (min:3,  max:4)、 QuotaB
```
Comment (denkensk) [inline kep/9-capacity-scheduling/README.md:266]:
```suggestion
We assume three elastic quotas are defined: QuotaA (min:3,  max:4), QuotaB
```

File: kep/9-capacity-scheduling/README.md [hunk 5]
```diff
@@ -262,6 +262,37 @@ guaranteed and hence non-preemptable.
...
+(min:4, max:6) and QuotaC (min:3,  max:4), the quota unit is the number of GPUs. UserA in blue color
+consumes QuotaA,UserB in red color consumes QuotaB and UserC in green color consumes QuotaC. The entire cluster
```
Comment (denkensk) [inline kep/9-capacity-scheduling/README.md:268]:
```suggestion
consumes QuotaA, UserB in red color consumes QuotaB and UserC in green color consumes QuotaC. The entire cluster
```

File: kep/9-capacity-scheduling/README.md [hunk 6]
```diff
@@ -262,6 +262,37 @@ guaranteed and hence non-preemptable.
...
+has 10 GPUs available hence the sum of quota min is equal to the cluster capacity.
+
+Initially, UserA consumes 2 GPUs,UserB consumes 2 GPUs and UserC consumes 3 GPUS. All consumptions
```
Comment (denkensk) [inline kep/9-capacity-scheduling/README.md:271]:
```suggestion
Initially, UserA consumes 2 GPUs, UserB consumes 2 GPUs and UserC consumes 3 GPUS. All consumptions
```

File: kep/9-capacity-scheduling/README.md [hunk 7]
```diff
@@ -262,6 +262,37 @@ guaranteed and hence non-preemptable.
...
+
+![6](./6.png)
+
+Later,UserB submits a pod that requests 1 GPU, at this time the cluster does not have enough resources and needs 
+to trigger a preemption. When `quota.used + preemptor.request <= min`, victims will be selected from the EQ which used > min
+In this case,`QuotaA.used > min`and `QuotaB.used <= min`.So a pod in QuotaA will be the victim.
+
+![7](./7.png)
+
+At some point,UserA consumes 2GPUs and UserC consumes 3 GPUs.UserB's two pods consume a total of 5 GPUs.
```
Comment (denkensk) [inline kep/9-capacity-scheduling/README.md:286]:
```suggestion
At some point, UserA consumes 2GPUs and UserC consumes 3 GPUs.UserB's two pods consume a total of 5 GPUs.
```

File: kep/9-capacity-scheduling/README.md [hunk 8]
```diff
@@ -262,6 +262,37 @@ guaranteed and hence non-preemptable.
...
+
+![8](./8.png)
+
+Next,UserB submits a TaskC that requests 1 GPU, at this time the cluster does not have enough resources and needs
```
Comment (denkensk) [inline kep/9-capacity-scheduling/README.md:290]:
```suggestion
Next, UserB submits a TaskC that requests 1 GPU, at this time the cluster does not have enough resources and needs
```

File: kep/9-capacity-scheduling/README.md [hunk 9]
```diff
@@ -262,6 +262,37 @@ guaranteed and hence non-preemptable.
...
+Initially, UserA consumes 2 GPUs,UserB consumes 2 GPUs and UserC consumes 3 GPUS. All consumptions
+are within the reserved "min" quota range.
+
+![5](./5.png)
+
+Next,UserA consumes 4 GPUs reaching the "max" quota of QuotaA.UserB consumes 3 GPUs.UserC still consumes 3 GPUs.
+
+![6](./6.png)
+
+Later,UserB submits a pod that requests 1 GPU, at this time the cluster does not have enough resources and needs
```
Comment (denkensk) [inline kep/9-capacity-scheduling/README.md:280]: we should keep the same expression `pod` or `task`

File: kep/9-capacity-scheduling/README.md (hunk 8 above)
Comment (denkensk) [reply kep/9-capacity-scheduling/README.md:290]: we should keep the same expression `pod` or `task`, pod is easy to be understand.

**Review by denkensk — COMMENTED** [review]
Thanks for your contribution. @yuchen-sun

File: kep/9-capacity-scheduling/README.md (hunk 1 above)
Comment (yuchen-sun) [reply kep/9-capacity-scheduling/README.md]: done

File: kep/9-capacity-scheduling/README.md (hunk 2 above)
Comment (yuchen-sun) [reply kep/9-capacity-scheduling/README.md]: done

File: kep/9-capacity-scheduling/README.md (hunk 3 above)
Comment (yuchen-sun) [reply kep/9-capacity-scheduling/README.md]: done

File: kep/9-capacity-scheduling/README.md (hunk 9 above)
Comment (yuchen-sun) [reply kep/9-capacity-scheduling/README.md:280]: done

**Comment by yuchen-sun** [comment]
Glad to help.
I solved all the issues that you suggested.
Please let me know if there is something else I can do.
@denkensk

**Comment by yuchen-sun** [comment]
When I was writing the controller logic, I found that the definition of crd does not match the one in kep.
Should I change here in this pr or do I need to open a new issue?
KEP:
```go
// ElasticQuotaStatus defines the observed use.
type ElasticQuotaStatus struct {
	Min  v1.ResourceList
	Max  v1.ResourceList
	Used v1.ResourceList
}
```
actually: [code](https://github.com/kubernetes-sigs/scheduler-plugins/blob/efe903ae7dd134a49def51a99c516f4592b56b8a/pkg/apis/scheduling/v1alpha1/types.go#L58)
```go
// ElasticQuotaStatus defines the observed use.
type ElasticQuotaStatus struct {
	// Used is the current observed total usage of the resource in the namespace.
	// +optional
	Used v1.ResourceList `json:"used,omitempty" protobuf:"bytes,1,rep,name=used,casttype=ResourceList,castkey=ResourceName"`
}
```

**Comment by denkensk** [comment]
lgtm
you can update the code in kep in this pr and please squash your commits. @yuchen-sun

**Comment by yuchen-sun** [comment]
> lgtm
> you can update the code in kep in this pr and please squash your commits. @yuchen-sun

Thanks!
I

**Comment by denkensk** [comment]
/assign @Huang-Wei 
ping again~ ^_^

**Comment by Huang-Wei** [comment]
Sry for the late response.

/approve

**Pull request merged**